src/mongocxx/pool.cpp

```diff
@@ -39,20 +39,20 @@ void pool::_release(client* client) {
 
 pool::~pool() = default;
 
-pool::pool(const uri& uri, stdx::optional<options::ssl> ssl_options)
+pool::pool(const uri& uri, const options::client& options)
```

Comment:
I can more easily imagine some future client option that can't be expressed in a URI than I can imagine a pool option.  But I don't have a very strong opinion either way.  All I really want  is some degree of symmetry between client/pool construction.